config/locales/en.yml

```diff
@@ -129,3 +132,4 @@ en:
   errors:
     messages:
       improbable_phone: 'Mobile number is invalid. Please make sure you enter a 10-digit phone number.'
+      invalid_confirmation_code: 'Invalid confirmation code.'
```

Comment:
Should we make this consistent with the other similar messages, such as adding `Please try again or request a new one`?